test/retrieve_test.rb

```diff
@@ -87,6 +87,46 @@ def test_retrieve_stuff
     assert_keys(export, keys, region: 'CA', date_number: dn)
   end
 
+  def test_all_keys
+    active_at = time_in_date('10:00', today_utc.prev_day(8))
+    two_days_ago = yesterday_utc.prev_day(1)
+
+    # Our retrieve endpoint returns keys SUBMITTED within the given period.
```

Comment:
Should we check the bounds? Like 14 days ago and today as well?

Reply:
Added!